.prettierignore

```diff
@@ -0,0 +1,9 @@
+dist
+dist-esm
+dist-es6
+dist-types
+
+modules/edit-modes/README.md
+modules/editor/README.md
+modules/layers/README.md
+modules/main/README.md
```

Comment:
why these files need to be ignored?

Reply:
I saw in gitignore that they are copied as part of the build process. So in the prettierignore i ignore everything in gitignore

https://github.com/uber/nebula.gl/blob/master/.gitignore#L21